Weather/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,10 @@
+/* 
+  Localizable.strings
+  Weather
+
+  Created by Alex on 6.09.22.
+  
+*/
+"Today" = "Today";
+"Temperature unit" = "°F";
```

Comment:
It's better to name your keys using camelCase: `today`, `temperatureUnit` and so on